### PR TITLE
bump architect-orb to 3.3.0

### PR DESCRIPTION
**Pull request by ubergesundheit**
Bump architect-orb

### CODE REVIEW

**Comment by MarcelMue** [comment]
Hey @ubergesundheit should this be updated / merged?

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 version: 2.1
 
 orbs:
-  architect: giantswarm/architect@2.11.0
+  architect: giantswarm/architect@dev:bump-app-build-suite
```
Comment (ubergesundheit) [inline .circleci/config.yml:5]:
```suggestion
  architect: giantswarm/architect@3.3.0
```

**Comment by ubergesundheit** [comment]
I would say this is optional, this is a leftover testing PR..

If CI works, it should be safe to merge though.

**Review by MarcelMue — APPROVED** [review]
Let's do it

**Pull request merged**